.gitignore

```diff
@@ -1,3 +1,5 @@
+.vscode/*
+ContainerHistoryLog
```

Comment:
@rushi47 not everyone is using vscode, you can add these files in a [global .gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) file on your system.

Reply:
@rst0git thanks for pointing out, will move in global